sample/sample.php

```diff
@@ -32,7 +38,7 @@
     ['timeout' => 5] // Guzzle HTTP request options
 ));
 
-if($token === null) {
+if ($token === null || $token === '') {
```

Comment:
How can it ever be `''` though? 🤔

Reply:
this was something I was testing and playing around with. Perhaps a left over from that 😄